source/mongodb-handshake/handshake.rst

```diff
@@ -224,6 +224,49 @@ Example::
         - "Oracle JVM EE 9.1.1"
 
 
+Supporting Wrapping Libraries
+=============================
+
+Drivers MUST allow libraries which wrap the driver to append to the client
+metadata generated by the driver. The following class definition defines the
+options which MUST be supported:
+
+.. code:: typescript
+
+  class DriverInfoOptions {
```

Comment:
Can we make the name of the class non-normative?  Java driver already defines it as MongoDriverInformation and it will be disruptive to change it now.

Reply:
I didn't assume it was, I thought most of our specs fall under the "acceptable deviations" clause of CRUD or SDAM. Do you want me to just omit the name completely? The language doesn't require you even to put it in a class, rather it says you need to support these options.

Reply:
Whatever works.  I think most specs that define an API repeat the acceptable deviations section.

Reply:
See my comment above, but I added a "Deviations" section to address this